### ISSUE TITLE
Can it record

### ISSUE THREAD

**Issue by steve55792** [NapoleonWils0n/pilfer#15]
Hi NapoleonWilson,

Can pilfer or rip-record  record this link: https://www.rapidvideo.com/e/FWZYQH1EH5 in Ubuntu?
  I seen in before discussions that can use a single quote  like: pilfer -i 'https://www.rapidvideo.com/e/FWZYQH1EH5'.

**Comment by NapoleonWils0n** [comment]
hi steve

no pilfer cant record that because its a web page

but youtube-dl can

heres how to download a single video

youtube-dl https://rapidvideo.com/e/FWZYQH1EH5


to batch download heres what you do

create a text file called links.txt

then add the rapidvideo links to links.txt file one url on each line
save the links.txt file

create a rapidvideo folder on your desktop and move the links.txt file inside the rapdvideo folder

open the terminal and change directory into the rapidvideo folder on your desktop like this

```
cd ~/Desktop/rapidvideo
```

type the ls command in the terminal and it should show the links.txt file

```
ls
```

now we can use youtube-dl with the -a option and the links.txt file
and it will dowload all the urls from the text file into the rapidvideo folder

```
youtube-dl -a links.txt
```

that will download all the urls rather than you doing each one by hand

**Comment by NapoleonWils0n** [comment]
hi steve

heres a list of sites youtube dl can download from

[youtube dl supported sites](https://rg3.github.io/youtube-dl/supportedsites.html)

**Comment by steve55792** [comment]
Thank you NapoleonWilson for the information, more the knowledge the better for me.

**Comment by NapoleonWils0n** [comment]
hi steve

no problem

**Comment by steve55792** [comment]
is it possible to set a time like 3pm today or so to start  a youtube-dl to download a link or pause or restart a link, This stuff is frickin interesting . Lol

**Comment by steve55792** [comment]
is it possible to set a time like 3pm today or so to start a youtube-dl to download a link or pause or restart a link, This stuff is frickin interesting . Lol  exexcuse me for closing this, i accidentally click while staring at my dog. Lol.

**Comment by NapoleonWils0n** [comment]
hi steve

you can use the at command to schedule starting youtube dl

https://github.com/NapoleonWils0n/kodi-playercorefactory/tree/master/docs/schedule-recordings

use youtube-dl and the url instead of the rip record command

you can also use cron jobs to schedule running scripts
youll need to google about that

im not sure about restarting youtube-dl 

you might find something in the youtube dl manual

```
man youtube-dl
```

use the arrow keys to move up and down the manual
and q to quit

**Comment by steve55792** [comment]
thank you

**Comment by NapoleonWils0n** [comment]
hi steve

another tip

you can get the url for the video from youtube-dl with the -g option like this

where theurl is the url you are trying to download

```
youtube-dl -g theurl
```

that displays the url in the terminal

you can save the url be redirecting the output to a text file like this

```
youtube-dl -g theurl > savedlink.txt
```

read the youtube-dl manual for more tips

```
man youtube-dl
```

**Comment by steve55792** [comment]
yep, i am reading the manual right now on the side. i gotta check if i can have it in a different format besides mp4, be cool if there's a dvd format so i dont have to convert.

**Comment by steve55792** [comment]
the youtube-dl format for windows is it the same as Ubuntu? the file txt i made is:
10 or more links into 1 text file name streamango. How would we set it up?the same as ubuntu?

**Comment by NapoleonWils0n** [comment]
hi steve

yes the format should be exactly the same on windows as on ubuntu

**Comment by NapoleonWils0n** [comment]
hi steve

-a is just a shortcut to --batch-file
they both do the same thing

but using -a is less to type 

so just use 

youtube-dl -a links.txt

and should work fine

**Comment by steve55792** [comment]
Sorry to keep bothering you on this  but i dont where i went wrong but here's a picture of it where streamango is the folder where the text file is at.
![capture](https://user-images.githubusercontent.com/192039/48740274-728eca00-ec1c-11e8-859f-a9c991f30cdf.PNG)

**Comment by NapoleonWils0n** [comment]
hi steve

no problem mate

im not a windows expert 

but i think it must be that the youtube-dl.exe isnt in your system path
but i have instructions on how to install youtube-dl from kodi playercofactory which should work

Download the youtube-dl exe here

[youtube-dl download](https://rg3.github.io/youtube-dl/download.html)

create a folder called youtube-dl and then inside it create a folder called bin

like this

youtube-dl/bin

copy the youtube-dl exe into youtube-dl\bin

then move the youtube-dl folder to your C drive

The file path should look like this

C:\youtube-dl\bin\youtube-dl.exe

then you need to add the path to the youtube-dl to your windows system path

Set up your windows path

system properties, advanced system settings, enviormnental variables

system variables, path, edit

just add this to the end of your windows system path and save

```
;C:\youtube-dl\bin
```

you may need to reboot and then hopefully windows will find youtube-dl in your windows system path

let me know if that works or you get stuck mate

**Comment by steve55792** [comment]
It works now,thank you again, in windows that is.

**Comment by NapoleonWils0n** [comment]
hi steve

great glad you managed to get it working

happy downloading
